rllib/algorithms/pg/pg.py

```diff
@@ -103,16 +102,16 @@ class PG(Algorithm):
 
     @classmethod
     @override(Algorithm)
-    def get_default_config(cls) -> AlgorithmConfigDict:
-        return PGConfig().to_dict()
+    def get_default_config(cls) -> AlgorithmConfig:
```

Comment:
Demonstrate how all Algorithms should look like in the future, only utilizing AlgorithmConfig as designed (not using the old-dict-helper-shims methods `__getattr__`, `__setattr__`, etc..).